### ISSUE TITLE
Warning on run ps command

### ISSUE THREAD

**Issue by mrqwer88** [major/MySQLTuner-perl#206]
Hello!

How I see it start reproduced after -   https://github.com/major/MySQLTuner-perl/pull/205
Part of output with problem - 

````
[--] Max MySQL memory    : 1.3G
Warning: bad ps syntax, perhaps a bogus '-'? See http://procps.sf.net/faq.html
[--] Other process memory: 26.3M
...
[OK] Maximum possible memory usage: 1.3G (4.18% of installed RAM)
Warning: bad ps syntax, perhaps a bogus '-'? See http://procps.sf.net/faq.html

```

In Debian 6 and Centos 6 (any another linux with procps version 3.2.8 ?)  old command work good, but new command work with warning  - 
```
# ps -eo pid,cmd >/dev/null
# ps -eaxo pid,cmd >/dev/null
Warning: bad ps syntax, perhaps a bogus '-'? See http://procps.sf.net/faq.html
```

**Comment by unreturned** [comment]
Such problem also in Debian 7

```
root@mail /tmp # perl mysqltuner.pl 
 >>  MySQLTuner 1.6.12 - Major Hayden <major@mhtx.net>
 >>  Bug reports, feature requests, and downloads at http://mysqltuner.com/
 >>  Run with '--help' for additional options and output filtering
[OK] Logged in using credentials from debian maintenance account.

[--] Skipped version check for MySQLTuner script
warning: bad ps syntax, perhaps a bogus '-'?
See http://gitorious.org/procps/procps/blobs/master/Documentation/FAQ
warning: bad ps syntax, perhaps a bogus '-'?
See http://gitorious.org/procps/procps/blobs/master/Documentation/FAQ
[OK] Currently running supported MySQL version 5.6.30
[OK] Operating on 64-bit architecture
 
-------- Storage Engine Statistics -----------------------------------------------------------------
[--] Status: +ARCHIVE +BLACKHOLE +CSV -FEDERATED +InnoDB +MEMORY +MRG_MYISAM +MyISAM +PERFORMANCE_SCHEMA 
[--] Data in InnoDB tables: 876M (Tables: 688)
[!!] Total fragmented tables: 3
 
-------- Security Recommendations ------------------------------------------------------------------
[OK] There are no anonymous accounts for any database users
[OK] All database users have passwords assigned
[!!] There is no basic password file list!
 
-------- CVE Security Recommendations --------------------------------------------------------------
[--] Skipped due to --cvefile option undefined
 
-------- Performance Metrics -----------------------------------------------------------------------
[--] Up for: 4m 5s (125K q [511.686 qps], 941 conn, TX: 158M, RX: 7M)
[--] Reads / Writes: 98% / 2%
[--] Binary logging is disabled
[--] Physical Memory     : 31.4G
[--] Max MySQL memory    : 5.4G
warning: bad ps syntax, perhaps a bogus '-'?
See http://gitorious.org/procps/procps/blobs/master/Documentation/FAQ
[--] Other process memory: 276.7M
[--] Total buffers: 5.1G global + 1.1M per thread (214 max threads)
[--] P_S Max memory usage: 78M
[--] Galera GCache Max memory usage: 0B
[OK] Maximum reached memory usage: 5.1G (16.38% of installed RAM)
[OK] Maximum possible memory usage: 5.4G (17.07% of installed RAM)
warning: bad ps syntax, perhaps a bogus '-'?
See http://gitorious.org/procps/procps/blobs/master/Documentation/FAQ
[OK] Overall possible memory usage with other process is compatible with memory available
[OK] Slow queries: 0% (0/125K)
[OK] Highest usage of available connections: 2% (5/214)
[OK] Aborted connections: 0.00%  (0/941)
[OK] Query cache is disabled by default due to mutex contention on multiprocessor machines.
[OK] Sorts requiring temporary tables: 0% (9 temp sorts / 51K sorts)
[OK] No joins without indexes
[OK] Temporary tables created on disk: 6% (33 on disk / 544 total)
[OK] Table cache hit rate: 45% (400 open / 875 opened)
[OK] Open file limit used: 0% (6/1K)
[OK] Table locks acquired immediately: 100% (98K immediate / 98K locks)
 
-------- ThreadPool Metrics ------------------------------------------------------------------------
[--] ThreadPool stat is disabled.
 
-------- Performance schema ------------------------------------------------------------------------
[--] Performance schema is enabled.
[--] Memory used by P_S: 78.9M
[--] Sys schema isn't installed.
 
-------- MyISAM Metrics ----------------------------------------------------------------------------
[!!] Key buffer used: 18.2% (3M used / 16M cache)
[OK] Key buffer size / total MyISAM indexes: 16.0M/101.0K
[OK] Read Key buffer hit rate: 100.0% (43K cached / 0 reads)
[OK] Write Key buffer hit rate: 100.0% (2K cached / 0 writes)
 
-------- AriaDB Metrics ----------------------------------------------------------------------------
[--] AriaDB is disabled.
 
-------- InnoDB Metrics ----------------------------------------------------------------------------
[--] InnoDB is enabled.
[OK] InnoDB buffer pool / data size: 4.0G/876.2M
[OK] InnoDB buffer pool instances: 4
[!!] InnoDB Used buffer: 21.22% (55634 used/ 262142 total)
[OK] InnoDB Read buffer efficiency: 100.00% (518135689 hits/ 518142258 total)
[!!] InnoDB Write Log efficiency: 59.14% (152 hits/ 257 total)
[OK] InnoDB log waits: 0.00% (0 waits / 409 writes)
 
-------- TokuDB Metrics ----------------------------------------------------------------------------
[--] TokuDB is disabled.
 
-------- Galera Metrics ----------------------------------------------------------------------------
[--] Galera is disabled.
 
-------- Replication Metrics -----------------------------------------------------------------------
[--] Galera Synchronous replication: NO
[--] No replication slave(s) for this server.
[--] This is a standalone server.
 
-------- Recommendations ---------------------------------------------------------------------------
General recommendations:
    Run OPTIMIZE TABLE to defragment tables for better performance
    MySQL started within last 24 hours - recommendations may be inaccurate
root@mail /tmp # cat /etc/issue
Debian GNU/Linux 7 \n \l

root@mail /tmp # 
```

**Comment by unreturned** [comment]
Also

```
root@mail /tmp # ps --version
procps-ng version 3.3.3
root@mail /tmp # ps -eo pid,cmd >/dev/null
root@mail /tmp # ps -eaxo pid,cmd >/dev/null
warning: bad ps syntax, perhaps a bogus '-'?
See http://gitorious.org/procps/procps/blobs/master/Documentation/FAQ
root@mail /tmp # 
```

problem exist after https://github.com/major/MySQLTuner-perl/commit/6d562f98b06dd9321c7040a8c4d9fe088df5a8f7

Thank you!

**Comment by iangregory** [comment]
Appears resolved in #209

**Comment by mrqwer88** [comment]
Thanks a lot!
